src/nativeresources/resourcestring.h

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#ifndef __RESOURCE_STRING_H_
+#define __RESOURCE_STRING_H_
+
+// Struct to contain a resource ID and its corresponding 
+// English language string.
+struct StringResource
```

Comment:
Can you please keep the "Native" prefix that it had before to make sure it is not mistaken for managed resources?